src/parser/validate.ts

```diff
@@ -163,31 +168,21 @@ export async function validate(parse: {
     return {...base, status: 'success'}
   }
 
-  async function validateRelationships(name: string, flag: CompletableFlag<any>): Promise<Validation[]> {
-    if (!flag.relationships) return []
-    const results = await Promise.all(flag.relationships.map(async relationship => {
-      const flags = relationship.flags ?? []
-      const results = []
+  function validateRelationships(name: string, flag: CompletableFlag<any>): Promise<Validation>[] {
+    return ((flag.relationships ?? []).map(relationship => {
```

Comment:
returns an array of promises.  No need to await them--someone else can do that later.